src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -237,6 +237,18 @@ public String getAwsSecretKey() {
     public String getAwsEndpoint() {
         return getString("aws.endpoint");
     }
+    
+    public boolean getAwsProxyEnabled(){
+    	return getBoolean("aws.proxy.isEnabled", false);
```

Comment:
Can you add a getBoolean method which doesn't require default value?  Or
directly calling mProperties.getBoolean()

On Wed, Feb 3, 2016 at 3:24 PM, Vivek <notifications@github.com> wrote:

> In src/main/java/com/pinterest/secor/common/SecorConfig.java
> <https://github.com/pinterest/secor/pull/175#discussion_r51805756>:
>
> > @@ -237,6 +237,18 @@ public String getAwsSecretKey() {
> >      public String getAwsEndpoint() {
> >          return getString("aws.endpoint");
> >      }
> > +
> > +    public boolean getAwsProxyEnabled(){
> > +    	return getBoolean("aws.proxy.isEnabled", false);
>
> for the boolean property, the method getBoolean expects 2 parameter,
> properyName and the defaultValue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pinterest/secor/pull/175/files#r51805756>.
>